webroot/results/admin/export_public.php

```diff
@@ -44,6 +44,10 @@
                               WHERE Competitions.showAtAll=1
                               GROUP BY Competitions.id',
     'Scrambles'   => '*',
+    'championships' => 'SELECT championships.competition_id, championships.championship_type
```

Comment:
Why would we exclude the `id` field here? I guess it seems useless, but you never know ^^ @jfly?

Reply:
I have a slight preference for keeping things as similar to the current database schema as possible, so I'd vote for including the id here.

Reply:
Nit: simplify to `SELECT championships.* FROM`, we don't want the competitions stuff =)